rosapi/scripts/rosapi_node

```diff
@@ -40,10 +40,12 @@ from rosapi.msg import *
 # Initialises the ROS node
 def register_services():
     rospy.Service('/rosapi/topics', Topics, get_topics)
+    rospy.Service('/rosapi/get_topics_types', TopicsTypes, get_topics_types)
```

Comment:
It is more sense to extend topics service to include type information instead of having extra service for both topic and types information. I would add string[] type in Topics.srv. And I would suggest to use MessageDetails.srv to get detailed message information for each topic.